.github/workflows/squash_on_demand.yml

```diff
@@ -0,0 +1,14 @@
+name: Softfix workflow
+on:
+  issue_comment:
+    types: [created]
+jobs:
+  softfix:
+    name: Softfix action
+    if: github.event.issue.pull_request != '' && contains(github.event.comment.body, '/softfix')
```

Comment:
How about using [`startsWith`](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions#startswith) instead of `contains`?